tests/mock_data/calicoctl/mesh/communities/input.yaml

```diff
@@ -13,11 +13,10 @@ spec:
     - cidr: 192.168.0.0/16
       communities:
       - bgp-comm-standard
-      - 5663:12  
```

Comment:
Test uses basic `diff` to compare expected and actual bird.cfg, as we are using map to store communities, order is not guaranteed and `diff` would fail so removing all except one community.